### PR TITLE
feat: add standard format to DateRangePickerField

### PR DESCRIPTION
**Pull request by Quis90**

### CODE REVIEW

File: src/Fields/DateRangePickerField.tsx [hunk 1]
```diff
@@ -33,7 +33,11 @@ export function DateRangePickerField<
   } = useController<TFieldValues, TName>(controller);
   return (
     <FieldWrapper controller={controller} formItem={formItem}>
-      <RangePicker format="DD.MM.YYYY" {...fieldProps} {...component} />
+      <RangePicker
+        format={['DD.MM.YYYY', 'DDMMYYYY']}
```
Comment (spawnia) [inline src/Fields/DateRangePickerField.tsx:37]: What does this change accomplish?

Comment (Quis90) [reply src/Fields/DateRangePickerField.tsx:37]: its adds 'DDMMYYYY' as possible format, but wait there is another change related to the rangepicker incoming

Comment (spawnia) [reply src/Fields/DateRangePickerField.tsx:37]: I don't understand how this impacts how this component works. I assume it only concerns data entry, but the component still canonicalizes the values it stores internally?

Comment (Quis90) [reply src/Fields/DateRangePickerField.tsx:37]: "When an array is provided, all values are used for parsing and first value is used for formatting" https://4x.ant.design/components/date-picker/#API

Comment (spawnia) [reply src/Fields/DateRangePickerField.tsx:37]: Nice, thank you!

**Pull request merged**